src/games/strategy/engine/framework/GameDataManager.java

```diff
@@ -260,40 +260,4 @@ private void writeDelegates(final GameData data, final ObjectOutputStream out) t
     out.writeObject(DELEGATE_LIST_END);
   }
 
-  /**
```

Comment:
Strip away the non test stuff and you are left with this as a starting point:
```
-      final GameDataManager manager = new GameDataManager();
 -      try {
 -        data = manager.loadGame(save);
 - } catch( IOException e ) {
 - }
```

- That is not much.
- We would be writing an integration test without any unit tests, it's a bad recipe of a starting point
- We are potentially going to rewrite how save game files are handled, as part of that we will likely write the desired test case while constructing that code. Testing the legacy stuff is not very valuable.